harvester_e2e_tests/apis/test_networks.py

```diff
@@ -99,3 +100,31 @@ def test_create_network_using_terraform(request, admin_session,
                                         harvester_api_endpoints,
                                         network_using_terraform):
     pass
+
+
+@pytest.mark.terraform
+def test_cluster_network_using_terraform(request, admin_session,
+                                         harvester_api_endpoints):
+
+    network_data = utils.create_clusternetworks_terraform(
+                       request,
+                       admin_session,
+                       harvester_api_endpoints,
+                       'resource_clusternetworks',
+                       'eth0')
```

Comment:
we shouldn't hardcode this to `eth0`, we should make it configurable and default to `harvester-mgmt` as that's the default bonded NIC name for the management network.

Reply:
fixed. this should come from --vlan-nic config